client/src/components/Tip/tests/Tip-story.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { setAddon, storiesOf } from '@storybook/react';
+import { withKnobs, selectV2, text } from '@storybook/addon-knobs';
+import { withNotes } from '@storybook/addon-notes';
+import JSXAddon from 'storybook-addon-jsx';
+
+setAddon(JSXAddon);
+
+import notes from '../README.md';
+
+import Tip, { TIP_IMPORTANCE_LEVELS } from 'components/Tip/Tip';
+import ValueTracker from 'stories/ValueTracker';
+import { Input, InputGroup, InputGroupAddon } from 'reactstrap';
+
+const inputProps = {
+  name: 'MyField',
+  id: 'MyField',
+  title: 'Field title',
+  placeholder: 'Placeholder text',
+};
+
+storiesOf('Admin/Tip', module)
+  .addDecorator(withKnobs)
+  .addDecorator((storyFn) => (
+    <ValueTracker>{storyFn()}</ValueTracker>
+  ))
+  .addWithJSX(
```

Comment:
e.g.:
```js
  .addWithJSX(
    'FormField example',
    withNotes(notes)(() => {
      const tip = {
        content: text('Content', 'Example tip contents'),
        icon: selectV2('Icon', ['lamp', 'flag', 'ticket']),
        importance: selectV2('Importance', importances, 'normal')
      };

      return <TextField tip={tip} />;
    }

  ));
```

Reply:
Hmm, I'm not sure this adds value - the JSX tab will output the `FieldHolder` component and nothing else, plus it'll look strange because there are no useful props (aside from the `tip`) passed to the `TextField`. I could reintroduce the 'With Tip' example in the TextField story though?

Reply:
It was more on the nice to have side of things.